Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/CheckInRetention.kt

```diff
@@ -0,0 +1,15 @@
+package de.rki.coronawarnapp.eventregistration.checkins
+
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.seconds
+import de.rki.coronawarnapp.util.TimeStamper
+import java.util.concurrent.TimeUnit
+
+private const val CHECK_IN_RETENTION_DAYS = 15
+private val CHECK_IN_RETENTION_SECONDS = TimeUnit.DAYS.toSeconds(CHECK_IN_RETENTION_DAYS.toLong())
+
+fun isWithinRetention(checkIn: CheckIn, timeStamper: TimeStamper): Boolean {
```

Comment:
Please add unit test for it? and maybe make it and extension for` checkIn`

`CheckIn. isWithinRetention(now)`

Reply:
Added.

Reply:
Making it an extension is a very good improvement, thanks!